README.md

```diff
@@ -1,46 +1,160 @@
+| `sympa_owner_priority`                     | `9`                                                                                                                   |                                                                                                                                                                                                                                                                                                                                          |
+| `sympa_default_list_priority`              | `5`                                                                                                                   | Default priority for list messages                                                                                                                                                                                                                                                                                                       |
+| `sympa_parsed_family_files`                | `[message.footer, message.header, message.footer.mome, message.header.mime, info ]`                                   | list of files that will be parsed by Sympa when  instantiating a family (no space allowed in file names)                                                                                                                                                                                                                                 |
+| `sympa_dkim_feature`                       | `"off"`                                                                                                               |                                                                                                                                                                                                                                                                                                                                          |
+| `sympa_dkim_add_signature_to`              | `[ robot, list ]`                                                                                                     | Insert a DKIM signature to message from the robot, from the list or both                                                                                                                                                                                                                                                                 |
+| `sympa_dkim_signature_apply_on`            | `[md5_authenticated_messages, smime_authenticated_messages, dkim_authenticated_messages, editor_validated_messages ]` | Type of message that is added a DKIM signature before distribution to  subscribers. Possible values are "none", "any" or a list of the following  keywords: "md5_authenticated_messages", "smime_authenticated_messages",  "dkim_authenticated_messages", "editor_validated_messages".                                                   |
```

Comment:
`none` and `all` are not possible with the role.
For `none` see my other comment, for `all` I don't think there is a way except for the user filling the list with the possible values. In that case I would remove the `all` option from the description.